packages/postgress/src/MessagePickupRepositoryPg.ts

```diff
@@ -0,0 +1,616 @@
+import {
+  AddMessageOptions,
+  Agent,
+  EncryptedMessage,
+  GetAvailableMessageCountOptions,
+  injectable,
+  Logger,
+  MessagePickupEventTypes,
+  MessagePickupLiveSessionRemovedEvent,
+  MessagePickupLiveSessionSavedEvent,
+  MessagePickupRepository,
+  QueuedMessage,
+  RemoveMessagesOptions,
+  TakeFromQueueOptions,
+} from '@credo-ts/core'
+import { Pool, Client } from 'pg'
+import PGPubsub from 'pg-pubsub'
+import * as os from 'os'
+import {
+  MessageState,
+  POSTGRES_PASSWORD,
+  POSTGRES_USER,
+  POSTGRES_HOST,
+  FCM_SERVICE_BASE_URL,
+} from '../config/constants'
+import { createTableMessage, tableNameLive, createTableLive } from '../config/dbCollections'
+import { ConnectionInfo } from './interfaces'
+import { MessagePickupSession } from '@credo-ts/core/build/modules/message-pickup/MessagePickupSession'
+import axios from 'axios'
+
+@injectable()
+export class MessagePickupRepositoryPg implements MessagePickupRepository {
+  private logger?: Logger
+  public messagesCollection?: Pool
+  private agent?: Agent
+  private pubSubInstance?: PGPubsub
+  private dbChannel?: string
```

Comment:
Is the dbChannel used anywhere? If it is useful for it to be configurable, we can probably put a field into the PostgresMessagePickupRepositoryConfig object (with some default value). Otherwise we can simply skip it and use a fixed value (which I think would make sense, provided that we are using a particular database for message pickup).

Reply:
Solved